src/shared/hooks/useAnalytics.ts

```diff
@@ -1,33 +1,25 @@
 import { useCallback, useState } from 'react'
-import PiwikTracker from '@amsterdam/piwik-tracker'
-import { UserOptions } from '@amsterdam/piwik-tracker/lib/types'
-import { usePiwik } from '@amsterdam/piwik-tracker-react'
+import PiwikPro, { PageViews } from '@piwikpro/react-piwik-pro'
 
-const rootUrl = process.env.REACT_APP_SELF_ROOT
-const siteId = process.env.REACT_APP_PIWIK_SITE_ID ?? ''
-const hasSiteId = !!process.env.REACT_APP_PIWIK_SITE_ID
+const dapUrl = 'https://dap.amsterdam.nl'
+const hasPiwikSiteId = !!process.env.REACT_APP_PIWIK_SITE_ID
```

Comment:
- Why the `?? ''`? Seems unnecessary
- Might it be clearer if we did: `hasPiwikSiteId = !!piwikSiteId`

Reply:
Shouldn't this be from an env?
const dapUrl = 'https://dap.amsterdam.nl'

Reply:
Aangepast